afkak/client.py

```diff
@@ -662,8 +669,13 @@ def _cancel_timeout(_, dc):
             # Set a delayedCall to fire if we don't get a reply in time
             dc = self._get_clock().callLater(
                 self.timeout, _timeout_request, broker, requestId)
-            # Setup a callback on the request deferred to cancel timeout
+            # Set a delayedCall to complain if the reactor has been blocked
+            rc = self._get_clock().callLater(
+                (self.timeout * 0.9), _alert_blocked_reactor, self.timeout,
+                time.time())
```

Comment:
Given what you do with this, I don't think that you need wall clock time. The elapsed time you can get from `IReactorTime.seconds()` is fine.

I vaguely recall seeing Glyph say that `seconds()` actually should be wall clock time somewhere. And practically speaking it probably couldn't be changed without breaking code, so it's probably not worth worrying about here. The only place I've seen it not be wall-clock time is the Z-series.